### DISCUSSION TITLE
Quad mesh generator?

### DISCUSSION THREAD

**Discussion by maryannewachter** [BHoM/BHoM#968] [Q&A]
I've seen that we have a wrapper for planar triangular meshing via the [TriangleNet_Toolkit](https://github.com/BHoM/TriangleNet_Toolkit) in the BHoM, but has there been any development on the quad mesh side suitable for floor slabs with complex geometry?

**Comment by IsakNaslundBh** [comment]
Had missed this, sorry. So far no development on an meshing in the triangle net toolkit for structural applications. Think the only thing done to date in there is Voronoi generation, basig triangulation for Env purposes and MeshRepresentations for BHoMObjects!

**Comment by al-fisher** [comment]
@maryannewachter you may be interested in the  `QuadRemesh` feature in Rhino 7
https://discourse.mcneel.com/t/rhino-7-feature-quadremesh/85601
https://www.rhino3d.com/features/quadremesh/
😄
